Makefile

```diff
@@ -106,11 +105,11 @@ install-staticcheck:
 
 .PHONY: install-goreleaser
 install-goreleaser:
-	$(call go-install-tool,$(GORELEASER),github.com/goreleaser/goreleaser@$(GORELEASER_VERSION),$(GORELEASER_VERSION))
+	$(call go-install-tool,$(GORELEASER),github.com/goreleaser/goreleaser/v2@$(GORELEASER_VERSION),$(GORELEASER_VERSION))
 
 .PHONY: install-syft
 install-syft:
-	$(call go-install-tool,$(SYFT),github.com/anchore/syft/cmd/syft@$(SYFT_VERSION),$(SYFT_VERSION))
+	curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- $(SYFT_VERSION)
```

Comment:
`curl | sh` is a dagnerous antipattern and should never be used.

Reply:
Can you make a contribution to syft? It's their recommended distribution method for their CLI tool.